doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,204 @@
+run for the updated file and see the result in the end.
+If you do not mount the `images` repo, you would have to
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
+`images` repo or all Pachyderm repositories as directories
+on you machine, edit as needed, and, when done,
+exit the `pachctl mount` command. Upon exiting the `pachctl mount`
+command, Pachyderm uploads all the changes to the corresponding
+repository.
+
+If someone else modifies the files while you are working on them
+locally, their changes will likely be overwritten when you exit
+`pachctl mount`. This happens because your changes are saved to
+the Pachyderm repository only after you interrupt the `pachctl mount`
+command. Therefore, make sure that you do not work on the
+same files while someone else is working on them.
+
+!!! warning
+    Use writable mount **ONLY** when you have sole ownership
+    over the mounted data. Otherwise, merge conflicts or
+    unexpected data overwrites can occur.
+
+Because output repositories are created by the Pachyderm
+pipelines, they are immutable. Only a pipeline
+can change and update files in these repositories. If you try to change
+a file in an output repo, you will get an error message.
+
+## Prerequisites
+
+You must have the following configured for this functionality to work:
+
+* Unix or Unix-like operating system, such as Ubuntu 16.04 or macOS
+Yosemite or later.
+* FUSE for your operating system installed:
+
+  * On macOS, run:
+
+    ```bash
+    brew cask install osxfuse
+    ```
+
+  * On Ubuntu, run:
+
+    ```bash
+    sudo apt-get install -y fuse
+    ```
+
+    For more information, see:
+
+    * [FUSE for macOS](https://osxfuse.github.io/)
+
+  !!! note
+      macOS has limited support for FUSE and might not be stable.
+
+## Mount a Pachyderm Repo
+
+Before you can mount a Pachyderm repo, verify that you have all the
+[Prerequisites](#prerequisites).
+
+To mount a Pachyderm repo on a local computer, complete the following
+steps:
+
+1. In a terminal, go to a directory in which you want to mount a
+Pachyderm repo. It can be any new empty directory on your local computer.
+For example, `pfs`.
+
+1. Run `pachctl mount` for a repository and branch that you want to mount:
+
+   ```bash
+   pachctl mount <path-on-your-computer> [flags]
+   ```
+
+   **Example:**
+
+   * If you want to mount all the repositories in your Pachyderm cluster 
+   to a `pfs` directory on your computer and give `WRITE` access to them, run:
+
+   ```bash
+   pachctl mount pfs --write
+   ```
+
+   * If you want to mount the master branch of the `images` repo
+   and enable file editing in this repository, run:
+
+   ```bash
+   pachctl mount pfs --repos images@master+w
+   ```
+
+   To give read-only access, omit `+w`.
+
+   **System Response:**
+
+   ```
+   ro for images: &{Branch:master Write:true}
```

Comment:
We might need someone with linux to try running through this and write the system responses. NO point in having all the mac-specific stuff here